### DISCUSSION TITLE
How to model a vertical rotor?

### DISCUSSION THREAD

**Discussion by arnal88** [petrobras/ross#922] [Q&A] [answered]
Hi,

I am currently working on vibration issues on a vertical axis machine, and I would like to model the rotor dynamics to understand what is going wrong.

I've just discovered the ROSS library and it seems to be exactly what I am looking for! 
Thank to all people who develop this outstanding tool.

However, I do not find in the documentation how to model a vertical axis. 
All examples are in a horizontal axis configuration, and it seems that the entire library is designed to model horizontal rotors only.

I think I could overcome this limitation by modeling my vertical rotor as horizontal and set gravity to zero.
But I do not find a way to set g=0 either.

So how could I model a vertical rotor using ROSS?

Thank you

**Comment by LeifParr** [comment, marked as answer]
Hi. I believe that the orientation of the axis only comes into play when calculating static load. For the dynamics, e.g. natural frequencies and damping, I do not believe that gravity is involved at all. So I think a starting point is to disregard static forces (e.g. solve them separately if needed), and use ross for the natural frequencies.

In most of my calculation, the rotors are vertical. The tricky part is to estimate the bearing forces in these machines (in my case hydropower units), which may range from very low to quite high.

**Comment by arnal88** [comment]
Hi LeifParr,

Many thanks for your reply.
I agree with you, axis orientation seems to come into play when calculating static load only.

I dug into the code and I eventually found how to set gravity to zero.
In the file "rotor_assembly.py" (line 2818):
`g = -9.8065`

So I set `g = 0` and I ran some dynamics analysis again: I got exactly the same results for frequencies, damping, etc.

Edit: I marked the discussion as "Answered"
